packages/amplify-e2e-tests/src/__tests__/uibuilder.test.ts

```diff
@@ -110,16 +110,28 @@ describe('amplify pull with uibuilder', () => {
       fs.readFileSync(path.join(__dirname, '..', 'cypress', 'uibuilder', 'uibuilder-spec.js')),
     );
 
-    const npmStartProcess = spawn(getNpmPath(), ['start'], { cwd: reactDir, timeout: 300000 });
-    // Give react server time to start
-    await new Promise((resolve) => setTimeout(resolve, 60000));
-    const res = execa.sync(getNpxPath(), ['cypress', 'run'], { cwd: reactDir, encoding: 'utf8' });
-    // kill the react server process
-    spawnSync('kill', [`${npmStartProcess.pid}`], { encoding: 'utf8' });
-    await new Promise((resolve) => setTimeout(resolve, 1000));
-    // Seriously, kill the react server process
-    // react-scripts somehow resurrects the process automatically after the first kill.
-    spawnSync('pkill', ['-f', 'react'], { encoding: 'utf8' });
-    expect(res.exitCode).toBe(0);
+    let npmStartProcess: ChildProcessWithoutNullStreams;
+    let res: execa.ExecaSyncReturnValue<string>;
+    describe('cypress tests', () => {
```

Comment:
Is this a `describe` nested inside an `it`? What purpose is this serving?

Reply:
See convo: https://github.com/aws-amplify/amplify-cli/pull/12799#discussion_r1230048525

Reply:
I did not notice that we're under `it`. I think npmStartProcess can happen in `it`. The killing should go to post-it. So should `describe` wrap `it` we're now nested under instead?